README.md

```diff
@@ -58,7 +58,9 @@ Add the following code to your page where appropriate. See the [Branded checkout
     show-cover-fees="true"
     on-order-completed="$event.$window.onOrderCompleted($event.purchase)"
     on-order-failed="$event.$window.onOrderFailed($event.donorDetails)"
-    radio-station-api-url="https://api.domain.com/getStations">
+    radio-station-api-url="https://api.domain.com/getStations"
+    radio-station-radius="100"
```

Comment:
This commit (`Allow branded checkout to configure hiding spouse information initially, and show it when button is clicked.`) is where you should rebase the radius squash into so it never happened.